public/gameInfo.php

```diff
@@ -60,7 +60,7 @@
 
 $pageTitle = "$gameTitle ($consoleName)";
 
-$relatedGames = getGameAlternatives($gameID);
+$relatedGames = getGameAlternatives($gameID, $sortBy);
```

Comment:
You probably don't want hubs to be rearranged on the game page when changing the sort order of achievements.  This needs to be behind a `isFullyFeaturedGame` check.